models/face_detection_yunet/README.md

```diff
@@ -19,13 +20,81 @@ Results of accuracy evaluation with [tools/eval](../../tools/eval).
 ## Demo
 
 Run the following command to try the demo:
+
 ```shell
 # detect on camera input
 python demo.py
 # detect on an image
 python demo.py --input /path/to/image
 ```
 
+#### Input
+
+Command: `--input /path/to/image`
+
+Usage: Set input to a certain image, omit if using camera.
+
+#### Model
+
+Command: `--model /path/to/model`
+
+Usage: Set model type, defaults to 'face_detection_yunet_2022mar.onnx'.
+
+#### Backend
+
+Command: `--backend (integers 0, 1, or 2)`
+
+Usage: Set backend DNN model, defaults to 'cv.dnn.DNN_BACKEND_OPENCV' (int = 0). Based on your OpenCV version, it may or may not support cv.dnn.DNN_BACKEND_TIMVX. More details: [https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f](https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f)
+
+| int | Model Type                |
+| --- | ------------------------- |
+| 0   | cv.dnn.DNN_BACKEND_OPENCV |
+| 1   | cv.dnn.DNN_BACKEND_CUDA   |
+| 2   | cv.dnn.DNN_BACKEND_TIMVX  |
+
+#### Target
+
+Command: `--target (integers 0, 1, 2, 3)`
+
+Usage: Set backend DNN model, defaults to 'cv.dnn.DNN_TARGET_CPU' (int = 0). Based on your OpenCV version, it may or may not support cv.dnn.DNN_TARGET_NPU. More details: [https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f](https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f)
+
+| int | Model Type                  |
+| --- | --------------------------- |
+| 0   | cv.dnn.DNN_TARGET_CPU       |
+| 1   | cv.dnn.DNN_TARGET_CUDA      |
+| 2   | cv.dnn.DNN_TARGET_CUDA_FP16 |
+| 3   | cv.dnn.DNN_TARGET_NPU       |
+
+#### Confidence Threshold
+
+Command: `--conf_threshold (float 0.01 to 1.00)`
+
+Usage: Set the minimum needed confidence for the model to identify a face, defauts to 0.9. Smaller values may result in faster detection, but will limit accuracy.
+
+#### NMS Threshold
+
+Command: `--nms_threshold (float 0.01 to 1.00)`
+
+Usage: Suppress bounding boxes of iou >= nms_threshold. Default = 0.3
+
+#### Top K
+
+Command: `--top_k (integer)`
+
+Usage: Keep top_k bounding boxes before NMS.
+
+#### Save Results
+
+Command: `--save (Boolean True or False)`
+
+Usage: Set “True” to save file with results (i.e. bounding box, confidence level). Invalid in case of camera input. Default will be set to “False”.
+
+#### Show Visualization Window
+
+Command: `--vis (Boolean True or False)`
+
+Usage: Default will be set to “True” and will open a new window to show results. Set to “False” to stop visualizations from being shown. Invalid in case of camera input.
+
```

Comment:
These documentations should be the help messages of arguments as follows:

https://github.com/opencv/opencv_zoo/blob/1ecf13e1deb97c9ae6a715ceb35553fb5eb37304/models/face_detection_yunet/demo.py#L34-L44

These arguments may be changed from time to time. If the documentation to these arguments are put in readme,  we will have to update readme if arguments are updated, which does not make sense to me. From my perspective, It is better to show the documentation to these arguments with `python demo.py --help`, which you can put along with the demo commands:
```.diff
# detect on camera input
python demo.py
# detect on an image
python demo.py --input /path/to/image
+ # show help messages
+ python demo.py --help
```

Reply:
Yes, that makes sense. Would you like me to make those changes and create a new commit?

Reply:
Yes, please do so.